test_scripts/API/LastMileNavigation/GetWayPoints/009_GetWayPoints_IN_USE_two_apps.lua

```diff
@@ -0,0 +1,105 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/25
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/embedded_navi/Get%20Destination_and_Waypoints.md
+-- Item: Use Case 1: Main Flow
+--
+-- Requirement summary:
+-- [GetWayPoints] As a mobile app I want to send a request to get the details of the destination 
+-- and waypoints set on the system so that I can get last mile connectivity.
+--
+-- Description:
+-- In case:
+-- 1) app2 sends second valid and allowed request during first one request from app1 is processing  on HMI 
+-- SDL must:
+-- 1) respond "IN_USE, success:false" to mobile application
```

Comment:
@LitvinenkoIra According to req-t both request should be processed successfully. 
Script has to be reworked as well.